definitions/contested/json/CaseEvent/CaseEvent.json

```diff
@@ -786,6 +786,7 @@
     "DisplayOrder": 122,
     "PostConditionState": "caseAdded",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/case-orchestration/contested/set-paper-case-defaults",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/case-orchestration/contested/set-paper-case-org-policy",
```

Comment:
any idea how this works now when it failed with the previous callback?

Reply:
It does set fields which are in event fields 
CallBackURLAboutToStartEvent can be used to save values
https://tools.hmcts.net/confluence/display/RCCD/Callbacks+%28Webhooks%29+documentation
"Can be used to pre-populate case data prior to submission."

I think the issue is it donesn't set items which are not in the caseEventToFields.
I did try adding paperapplication, isAdmin etc to the fields but hiding them from the screen but it didn't take. I'm thinking they might need to be displayed.
This was the quick workaround which sorts it, though in theory its doable the other way